worker/include/Channel/Request.hpp

```diff
@@ -67,7 +67,7 @@ namespace Channel
 		static std::unordered_map<std::string, MethodId> string2MethodId;
 
 	public:
-		Request(Channel::UnixStreamSocket* channel, json& jsonRequest);
+		Request(json& jsonRequest);
```

Comment:
The constructor declaration must have `explicit` keyword in front (since it has a single argument).

Reply:
agree. already add `explicit` keyword.